code/modules/research/destructive_analyzer.dm

```diff
@@ -91,7 +91,7 @@ Note: Must be placed within 3 tiles of the R&D Console
 			loaded_item = S
 		else
 			qdel(S)
-	else
+	if(thing)

```

Comment:
Shouldn't this be `else if`? Otherwise whole stacks will be deleted

Reply:
It does, but the stack is being reclaimed without being clearing.
I might be misunderstanding how reclaim works.
Should they take a loss on a stack? or should it just take several reclaims to take all of it?

Reply:
It looks like the code above explicitly takes one item from the stack

Reply:
I now see what you mean, i have redone sheet processing and fixed a second part of the issue that was not reported.
If you had 10 sheets in the destro and reclaimed mats it would remove one sheet, and give you the mats for 10 sheets.
Does the new version resolve this?